### PR TITLE
version 1.2.12 for PhpStorm 2024.1

### PR DESCRIPTION
**Pull request by Danil42Russia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [1.2.12] - 09.04.2024
+
+- adapt code for 2023.2
```
Comment (Hidanio) [inline CHANGELOG.md:7]: 2023?

**Review by Hidanio — CHANGES_REQUESTED** [review]

File: gradle.properties [hunk 2]
```diff
@@ -5,23 +5,23 @@ pluginName = kphpstorm
...
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
 # PHP Plugin - https://plugins.jetbrains.com/plugin/6610-php/versions
-platformPlugins = com.jetbrains.php:233.11799.232
+platformPlugins = com.jetbrains.php:241.14494.237
 
 # Gradle Releases -> https://github.com/gradle/gradle/releases
-gradleVersion = 8.5
+gradleVersion = 8.6
```
Comment (Hidanio) [inline gradle.properties:24]: maybe 8.7?

**Review by Hidanio — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 2 above)
Comment (Danil42Russia) [reply gradle.properties:24]: No. Let's not update earlier than we did in [intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)

Comment (Danil42Russia) [reply gradle.properties:24]: No. Let's not update earlier than we did in [intellij-platform-plugin-template](https://github.com/JetBrains/intellij-platform-plugin-template)

File: CHANGELOG.md (hunk 1 above)
Comment (Danil42Russia) [reply CHANGELOG.md]: done

**Review by Hidanio — APPROVED** [review]

**Pull request merged**